ocs_ci/utility/utils.py

```diff
@@ -2661,11 +2662,15 @@ def create_rhelpod(namespace, pod_name, timeout=300):
 
     label_pod_security_admission(namespace=namespace)
 
-    # TODO: This method should be updated to add argument to change RHEL version
+    if rhel_version >= 8:
+        RHEL_POD_YAML = constants.RHEL_8_7_POD_YAML
+    else:
+        RHEL_POD_YAML = constants.RHEL_7_7_POD_YAML
+
```

Comment:
I think that the `rhel_pod_yaml` variable should be lowercase:
```suggestion
    if rhel_version >= 8:
        rhel_pod_yaml = constants.RHEL_8_7_POD_YAML
    else:
        rhel_pod_yaml = constants.RHEL_7_7_POD_YAML
```

Reply:
done